dashboards/lodestar_vm_host.json

```
@@ -1030,7 +941,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "editorMode": "code",
-          "expr": "nodejs_eventloop_lag_seconds",
+          "expr": "nodejs_eventloop_lag_seconds{job=\"beacon\"}",
```

Job name variable needs to be supported, else job name is not configurable

```suggestion
          "expr": "nodejs_eventloop_lag_seconds{job=~\"$beacon_job|beacon\"}",
```

**Edit:** Updated lint script to ensure this ☝️

---

Good point! We should add a lint rule for it